docs/src/pages/customization/css-in-js.md

```diff
@@ -92,6 +92,10 @@ You might, however, also want to override these styles, for example with styled-
 If you are experiencing a CSS injection order issue, JSS [provides a mechanism](https://github.com/cssinjs/jss/blob/master/docs/setup.md#specify-dom-insertion-point) to handle this situation.
 By adjusting the placement of the `insertionPoint` comment within your HTML body you can [control the order](http://cssinjs.org/js-api/#attach-style-sheets-in-a-specific-order) that the CSS rules are applied to your components.
 
+### HTML comment
+
+The simplest approach:
```

Comment:
"The simplest approach is to add an HTML comment that determines where JSS will inject the styles:"